data/autoyast_sle15/autoyast_sle-micro_updates.xml.ep

```diff
@@ -5,7 +5,6 @@
     <do_registration config:type="boolean">true</do_registration>
     <reg_code>{{SCC_REGCODE}}</reg_code>
     <install_updates config:type="boolean">true</install_updates>
-    <reg_server>{{SCC_URL}}</reg_server>
```

Comment:
Why is this being removed? Is the registration now being done after the installation?

Reply:
no, SCC_URL is not used in updates testing as we cannot use proxy SCC (developemnt thingy). Mainly the variable is not even defined for update mediums.

Reply:
http://kepler.suse.cz/tests/19436#step/bootloader_start/9 see `regurl`